lwreg/test_lwreg.py

```diff
@@ -593,7 +593,7 @@ def testStandardizeMolFunction(self):
         nm = utils.standardize_mol(m, config=cfg)
         self.assertEqual(Chem.MolToSmiles(nm), 'CCO')
 
-
+"""
```

Comment:
Please just remove this.
It's not doing anything and might as well be totally removed

Reply:
I think it's better to just delete the code than it is to leave in something that's commented out.

(General style aside: it's clearer to comment stuff out with `#` rather than putting it in a giant string)

Reply:
Apologies, that one slipped. Removed now